### ISSUE TITLE
No more shiny pokemon

### ISSUE THREAD

**Issue by SDB-H** [iam4x/pokemongo-webspoof#470]
I've been using this (with the custom script to overcome the freeze issue), but now it seems like I'm banned from finding shinies. I have more than 4000 catches without a single shiny over the last weeks, anyone else still using this noticed the same?
Anyone has recommendations that are viable alternatives to this tool?

**Comment by maxkuhlmay** [comment]
This is not quite related to this app, right?
This app just fakes the gps location of your iPhone, so you can move in Pokemon Go.

Im not quite sure if Pokémon go can detect this app and ban you. If so, you should be able to catch any Pokemon.

It might not be a ban, just bad luck.

**Comment by SDB-H** [comment]
True, it's not directly related to the app, although I could envision a scenario where Niantic started to include height information to do the RNG for shinies. Bad luck is hard to exclude, I was just curious whether or not others were experiencing the same to back up my hypothesis.
